sdk/harambe/core.py

```diff
@@ -147,9 +148,11 @@ async def enqueue(
         :param urls: urls to enqueue
         :param context: additional context to pass to the next run of the next stage/url
         :param options: job level options to pass to the next stage/url
+        :param stage: the override stage to use for the next job. Will use the next stage in the sequence if not provided.
         """
         context = context or {}
         options = options or {}
+        stage = stage or get_next_stage(self._stage)
```

Comment:
Would recommend leaving the getnextstage outside of harambe (separation of concerns)